website/docs/reference/widgets/table/README.md

```diff
@@ -103,6 +103,30 @@ This method uses a value in the API response as the key to the following API cal
 
 <img src="/img/pagination_(2)_(2).gif" width="70%" ></img>
 
+#### Meta properties based pagination
+
+Table widget exposes two meta properties, `previousPageVisited` and `nextPageVisited`. These values of these properties are set when a user moves to a new page. Depending on the direction of the new visited page with respect to the current page, the values of these properties are set.
+
+If the page number of the new visited page is greater than the current page number, `nextPageVisited` will be set to `true` and `previousPageVisited` will be set to `false`. Else, if the new page number is less than current page number, `nextPageVisited` will be set to `false` and `previousPageVisited` will be set to `true`.
+
+Both flags are initialized to `false` when the table loads for the first time.
+
+A sample code that demonstrates the use of these flags is below:
+
+```
+const loadTableData =  () => {
```

Comment:
From the issue, should we add context on why we made this change to the docs itself. This context will be lost for the users otherwise. Only a suggestion, thinking out loud

Reply:
Please refer to @dilippitchika's comment below : https://github.com/appsmithorg/appsmith-docs/pull/1144#issuecomment-1512414451